### PR TITLE
Expanded permission_ethereumAccounts spanish translation

### PR DESCRIPTION
**Pull request by lucasgday**
Reflected complete message from the [english "message"](https://github.com/MetaMask/metamask-extension/blob/1772deeea9929579b8fc4425f414f75802070798/app/_locales/en/messages.json#L2576): "See address, account balance, activity and suggest transactions to approve".

## Explanation

* What is the current state of things and why does it need to change?
* What is the solution your changes offer and how does it work?

Currently, the spanish message doesn't represent accurately the `eth_accounts` permission

This is a problem because it's not clear what users are enabling.

In order to solve this problem, this pull request expands the translation of this line to explain this permission completely.

## More Information

* PR [#13605](https://github.com/MetaMask/metamask-extension/pull/13605/files/c62191b6be6951b0a4a455e3a60df1a3a67bf4ad)

## Manual Testing Steps

- Configure MetaMask language to Spanish
- Go to a website that needs to connect to MetaMask Wallets for read permissions.
- Start connection process
- Select Accounts to Connect
- Read the Message from MetaMask popup

## Pre-Merge Checklist

- [X] PR template is filled out
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [X] PR is linked to the appropriate [GitHub issue](https://github.com/MetaMask/metamask-extension/issues/15659)
- [X] PR has been added to the appropriate release Milestone #15661

### CODE REVIEW

**Comment by lucasgday** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by bschorchit** [comment]
@seaona might be able to help review this :)

File: app/_locales/es/messages.json [hunk 1]
```diff
@@ -2472,7 +2472,7 @@
     "description": "The description for the `snap_confirm` permission"
   },
   "permission_ethereumAccounts": {
-    "message": "Ver las direcciones de las cuentas permitidas (requerido)",
+    "message": "Ver las direcciones, balances, actividad de las cuentas permitidas y sugerir transacciones para aprobar (requerido)",
```
Comment (seaona) [inline app/_locales/es/messages.json:2475]: I would change `balances` for `saldos` - using the Spanish translation would be more consolidated as it is already used in the rest of the Extension

**Review by seaona — CHANGES_REQUESTED** [review]
I would change `balances` for `saldos` - using the Spanish translation would be more consolidated as it is already used in the rest of the Extension

**Comment by lucasgday** [comment]
Thanks for the review @seaona ! I just commited the change. Lmk if there's anything else I should do!

**Review by seaona — APPROVED** [review]

**Comment by seaona** [comment]
@lucasgday looks good! Thank you for the change :)

**Comment by lucasgday** [comment]
Hi @seaona! What's the status for this change?

**Review by brad-decker — APPROVED** [review]

**Comment by brad-decker** [comment]
@lucasgday sorry about that, we dropped the ball. I am updating your branch and assuming everything is green i'll merge today.

**Pull request merged**